src/components/common/RuleListItem.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <li class="rule-list-item">
```

Comment:
In most components, we'd use an `id` here, but there may be several instances of this component on a page, and ids are supposed to be unique, so I'm using a classname here instead.